bbb-activity-report/activity-report.nginx

```diff
@@ -0,0 +1,4 @@
+location /activity-report {
+        root    /var/bigbluebutton;
```

Comment:
Please instead of `root` use `alias` ( so you can specify the full path ):
`alias  /var/bigbluebutton/activity-report/;`

Also, add this, so directories are not browseable:
`autoindex off;`